packages/mint-components/src/components/sqm-share-code/useShareCode.tsx

```diff
@@ -24,11 +25,24 @@ const MessageLinkQuery = gql`
 `;
 
 export function useShareCode(props: ShareCodeProps): ShareLinkViewProps {
-  const { programId = useProgramId() } = props;
+  const programId = useDomContext("sq:program-id");
```

Comment:
Hmm, why can't this `useProgramId`? Why does it have to use the context directly?

Reply:
this is incomplete 🙈